perk-store-webapps/src/main/webapp/vue-app/components/PerkStoreApp.vue

```diff
@@ -360,17 +278,23 @@ import ProductNotification from './perk-store/ProductNotification.vue';
 import {initSettings, getOrderFilter, getProductFilter, storeProductFilter} from '../js/PerkStoreSettings.js';
 import {toFixed} from '../js/PerkStoreProductOrder.js';
 import {getProductList, getProduct} from '../js/PerkStoreProduct.js';
+import CreateProductButton from './perk-store/CreateProductButton.vue';
```

Comment:
This is a bad practice to declare an import inside a vue file.
As made in other addons, you should update this module by defining initComponents.js where you can import and declare the Vue files.

Reply:
done !